content/hi/docs/reference/glossary/daemonset.md

```diff
@@ -0,0 +1,19 @@
+---
+title: डेमनसेट (DaemonSet)
+id: daemonset
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/daemonset
+short_description: >
+ यह सुनिश्चित करता है कि क्लस्टर में नोड्स के सेट पर पॉड की एक प्रतिलिपि चल रही है।
+aka:
+tags:
+  - fundamental
+  - core-object
+  - workload
+---
+
+यह सुनिश्चित करता है कि {{< glosary_tooltip text="क्लस्टर" term_id="cluster" >}} में नोड्स के सेट पर {{< glosary_tooltip text="पॉड" term_id="pod" >}} की एक प्रतिलिपि चल रही है।
```

Comment:
Misspelt my suggestion of glossary while suggesting the change last time, apologies for that. Thank you @tengqm for pointing this out.
```suggestion
यह सुनिश्चित करता है कि {{< glossary_tooltip text="क्लस्टर" term_id="cluster" >}} में नोड्स के सेट पर {{< glossary_tooltip text="पॉड" term_id="pod" >}} की एक प्रतिलिपि चल रही है।
```